apps/backend/src/db/migrations/20240910113839_add_tasks_table.ts

```diff
@@ -0,0 +1,64 @@
+import { type Knex } from "knex";
+
+const TableName = {
+	CATEGORIES: "categories",
+	TASKS: "tasks",
+	USER: "users",
+} as const;
+
+const ColumnName = {
+	CATEGORY_ID: "category_id",
+	CREATED_AT: "created_at",
+	DESCRIPTION: "description",
+	DUE_DATE: "due_date",
+	ID: "id",
+	LABEL: "label",
+	STATUS: "status",
+	UPDATED_AT: "updated_at",
+	USER_ID: "user_id",
+} as const;
+
+const TaskStatus = {
+	COMPLETED: "Completed",
+	CURRENT: "Current",
+	SKIPPED: "Skipped",
+} as const;
+
+const DELETE_STRATEGY = "CASCADE";
+
+async function up(knex: Knex): Promise<void> {
+	await knex.schema.createTable(TableName.TASKS, (table) => {
```

Comment:
When adding a migration with a new table, you should also add the table to the diagram in README.md.